src/main/webapp/app/communication/posting-create-edit-modal/post-create-edit-modal/post-create-edit-modal.component.html

```
@@ -1,54 +1,61 @@
-                <div class="row mb-2">
-                    <jhi-posting-markdown-editor
-                        formControlName="content"
-                        [editorHeight]="editorHeight"
-                        [maxContentLength]="maxContentLength"
-                        [suppressNewlineOnEnter]="false"
-                        [isButtonLoading]="isLoading"
-                        [isFormGroupValid]="formGroup.valid"
-                        [editType]="editType"
-                    />
-                </div>
+@if (isDialogVisible) {
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)